test/helpers/envtest.go

```diff
@@ -141,23 +142,23 @@ func NewTestEnvironment() *TestEnvironment {
 		Password:   simr.Password(),
 	}
 	managerOpts.AddToManager = func(ctx *context.ControllerManagerContext, mgr ctrlmgr.Manager) error {
```

Comment:
Maybe let's add all webhooks. I think only VSphereClusterTemplateWebhook is missing

Reply:
Fixed.